test/tests_for_auth/ee2_admin_mode_test.py

```diff
@@ -109,31 +114,40 @@ def get_runner_with_condor(self) -> SDKMethodRunner:
 
     # TODO How do you test ADMIN_MODE without increasing too much coverage
 
-    def get_mocks(
-        self, user_id=None, token="fake_token"
+    def get_user_mocks(
+        self, user_id=None, token=None
     ) -> (UserClientSet, Workspace, WorkspaceAuth):
         user_id = user_id if user_id else self.user_id
+        token = token if token else self.token
         ws = create_autospec(Workspace, instance=True, spec_set=True)
         wsa = create_autospec(WorkspaceAuth, instance=True, spec_set=True)
         ucs = UserClientSet(user_id, token, ws, wsa)
         return ucs, ws, wsa
 
+    def get_client_mocks(self, *to_be_mocked):
+        return _get_client_mocks(self.cfg, self.config_file, *to_be_mocked)
+
     @patch.object(Catalog, "get_module_version", return_value="module.version")
```

Comment:
Reworking all the patching in ee2 would be a monumental tasks and would blow this PR up. This is something that can be done as necessary.